runtime/onert/core/include/exec/Execution.h

```diff
@@ -158,6 +158,14 @@ class Execution
    * @return @c float Loss value
    */
   float getLoss(const ir::IOIndex &ind);
+
+  /**
+   * @brief     Export circle
+   * @note      It should be called after training
+   * @param[in] path   Output path
+   * @throw     runtime_error on wrong path or permission denied
+   */
+  void export_circle(const std::string &path) const;
```

Comment:
Note that it is enclosed with `#ifdef ONERT_TRAIN`. `#ifdef ONERT_TRAIN` should be removed.